### PR TITLE
Revert to appropriate definition generation

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: utils/manifests/definitions.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ type Component struct {
 
 type Config struct {
 	Name            string                 // Name of the service mesh,or k8 or meshery
+	Type            string                 //Type of the workload like- Istio, TraefikMesh, Kuma, OSM, Linkerd,AppMesh,NginxMesh
```
Comment (leecalcote) [inline utils/manifests/definitions.go:22]: @Revolyssup "Type" intending to serve the purpose of "registrant"?

Comment (Revolyssup) [reply utils/manifests/definitions.go:22]: No this is not registerant. That will be added in adapter library

Comment (Revolyssup) [reply utils/manifests/definitions.go:22]: You can go ahead and merge this

Comment (leecalcote) [reply utils/manifests/definitions.go:22]: 👍

**Pull request merged**